extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -288,17 +294,40 @@ public function getToolbarTemplate() {
 	/**
 	 * Link for adding new section on script talk page. Prefilled with standard explanation of rejection.
 	 * @param \Title $title Title object of JS page
+	 * @param int $revisionId
 	 * @return string full link to edit page
 	 */
-	public function prepareProvideFeedbackLink( \Title $title ) {
+	public function prepareProvideFeedbackLink( \Title $title, $revisionId = 0 ) {
 		$params = [
 			'action' => 'edit',
 			'section' => 'new',
-			'useMessage' => 'content-review-rejection-explanation'
+			'useMessage' => 'content-review-rejection-explanation',
 		];
+
+		if ( (int)$revisionId !== 0 ) {
+			$params['messageParams'] = [
+				1 => wfMessage( 'content-review-rejection-explanation-title' )->params( $revisionId )->escaped(),
+				2 => $title->getFullURL( "oldid={$revisionId}" ) . " {$revisionId}",
```

Comment:
Was thinking about it but did not want to "overengineer" it. However, if you too find it reasonable, I am going to do it. Thanks! :)